internal/cliutils/checkforupdate.go

```diff
@@ -27,7 +29,9 @@ func CheckForUpdate() (*string, error) {
 
 func UpdateFetch() {
 	newerVersion, err := CheckForUpdate()
-	if err == nil && newerVersion != nil {
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "There was an error checking for updates: %v\n", err)
```

Comment:
I think the check should continue to fail silently, as it is not really related to what the user is doing